spec/reservation_tracker_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'date'
+require 'spec_helper'
```

Comment:
My tests wouldn't run unless I changed this to `require_relative` -- How did you get your tests to run?